pkg/devcontainer/docker_client.go

```diff
@@ -0,0 +1,36 @@
+package devcontainer
+
+import (
+	"context"
+	"io"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+type DockerClient interface {
```

Comment:
I think we should just use [this](https://github.com/moby/moby/blob/master/client/interface.go#L48)

Reply:
Done